doc/docker.md

```diff
@@ -1,16 +1,102 @@
 # Running with Docker
 
+The advantage of using docker-compose is that it is configured to run several services that the Control Panel relies on, such as the PostgreSQL database and Redis. It initializes these ready for local development purposes.
+
+## Build
+
 To build the docker image, use the following command:
+
 ```sh
 docker build -t controlpanel .
 ```
 
-Running the app requires a PostgreSQL database, and this is provided with
-docker-compose:
+## .env file
+
+You need to create a `.env` file with the settings to enable it to connect to external services.
+
+You probably want it to pick up your 'data' AWS profile:
```

Comment:
What's a `data` AWS profile and how do I get one..? (Please provide a link to this information). 

(NB: as a new joiner I probably know what an AWS profile is, but I don't know whatthe right settings are to use with the `aws` command. Furthermore, *I* (ntoll) use the name `moj` for my Amazon profile for MOJ things and the `data` name for the profile feels a bit arbitrary - i.e. why not say something like, _"You probably want it to pick up your MoJ AWS profile (in the following example, called `data`)"_..? See, for instance, [these instructions](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/master/doc/running.md#aws-configuration))

Reply:
I've now written instructions for this in the -iam repo and added a link here